### DISCUSSION TITLE
Operator Installation per Namespace via OLM

### DISCUSSION THREAD

**Discussion by tobru** [appuio/appuio-cloud-community#7] [Ideas]
Initially proposed by @christophbrejla:

> I propose the idea of unlocking the feature for users to install operators for their namespaces (https://docs.openshift.com/container-platform/4.9/operators/user/olm-installing-operators-in-namespace.html)
> 
> in my case i would possibly try the operators for 
> 
> - bitnami sealed secrets
> - prometheus
> - dynatrace

**Comment by ghost** [comment]
Another interesting operator is the [Keycloak Operator](https://operatorhub.io/operator/keycloak-operator) to have a authentication service for web applications.
